src/routes/townhalls/rating/index.ts

```diff
@@ -0,0 +1,24 @@
+import { Router } from 'express';
+import Joi from 'joi';
+import type { RatingForm } from 'prytaneum-typings';
+import { ratingValidationObject } from 'modules/townhall/validators';
+import API from 'modules/rating';
+
+import { makeJoiMiddleware, makeEndpoint } from 'middlewares';
+import { TownhallParams, RatingParams } from '../types';
+
+const router = Router();
+
+router.put<TownhallParams, void, RatingForm, RatingParams>(
+    '/:townhallId/rating',
```

Comment:
Keep the collection name consistent with the route name. `Ratings` is a sub-collection of `Townhalls`.  Therefore, the full url should look like this: /api/townhalls/123/__ratings__